### PR TITLE
FlashControl.Auto now accurately describes how it affects the focus assist light

### PR DESCRIPTION
**Pull request by vtest-ms**

### CODE REVIEW

**Comment by ktoliver** [comment]
Hi @drewbatgit - Could you review the proposed updates to your articles? The new links should be revised to relative-link format and remove locale before the PR is merged.

File: windows.media.devices/flashcontrol_assistantlightenabled.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ public bool AssistantLightEnabled { get;  set; }
 Gets or sets a value indicating whether focus assist light is enabled on the capture device.
 
 ## -property-value
-A value indicating whether focus assist light is enabled on the capture device.
+A value indicating whether focus assist light is enabled on the capture device. If [Auto](https://docs.microsoft.com/en-us/uwp/api/windows.media.devices.flashcontrol.auto?view=winrt-19041#Windows_Media_Devices_FlashControl_Auto) is enabled in addition to this, then the focus assist light will behave automatically as well.
```
Comment (kasumman) [inline windows.media.devices/flashcontrol_assistantlightenabled.md]: >https://docs.microsoft.com/en-us/uwp/api/windows.media.devices.flashcontrol.auto?view=winrt-19041#Windows_Media_Devices_FlashControl_Auto [](start = 90, length = 137)

change this to just 
flashcontrol_auto.md #Closed

File: windows.media.devices/flashcontrol_auto.md [hunk 2]
```diff
@@ -10,10 +10,10 @@ public bool Auto { get;  set; }
 # Windows.Media.Devices.FlashControl.Auto
 
 ## -description
-Gets or sets a value that specifies if flash is automatically set.
+Gets or sets a value that specifies if flash is automatically set. This value also determines if the focus assist light operates automatically or by default whenever [AssistantLightEnabled](https://docs.microsoft.com/en-us/uwp/api/windows.media.devices.flashcontrol.assistantlightenabled?view=winrt-19041#Windows_Media_Devices_FlashControl_AssistantLightEnabled) is set.
```
Comment (kasumman) [inline windows.media.devices/flashcontrol_auto.md]: >https://docs.microsoft.com/en-us/uwp/api/windows.media.devices.flashcontrol.assistantlightenabled?view=winrt-19041#Windows_Media_Devices_FlashControl_AssistantLightEnabled [](start = 190, length = 171)

use relative link: flashcontrol_assistantlightenabled.md #Closed

File: windows.media.devices/flashcontrol_auto.md [hunk 3]
```diff
@@ -10,10 +10,10 @@ public bool Auto { get;  set; }
...
 
 ## -property-value
-**true** if auto flash is set; otherwise, **false**.
+**true** if auto flash is set; otherwise, **false**. If this value is **true** while [AssistantLightEnabled](https://docs.microsoft.com/en-us/uwp/api/windows.media.devices.flashcontrol.assistantlightenabled?view=winrt-19041#Windows_Media_Devices_FlashControl_AssistantLightEnabled) is also **true**, then the focus assist light will operate automatically instead of by default.
```
Comment (kasumman) [inline windows.media.devices/flashcontrol_auto.md]: >https://docs.microsoft.com/en-us/uwp/api/windows.media.devices.flashcontrol.assistantlightenabled?view=winrt-19041#Windows_Media_Devices_FlashControl_AssistantLightEnabled [](start = 109, length = 171)

same here #Closed

**Review by kasumman — APPROVED** [review]
:shipit:

**Comment by ktoliver** [comment]
Hi @kasumman - Thanks for your review! The PR can be merged after a member of the content team or a PM indicates approval. 

@drewbatgit - Could you take a look at the final version of the PR? Thanks.

**Comment by lthemep** [comment]
how to set focus assist light auto and flash enable case?

**Review by drewbatgit — APPROVED** [review]

**Pull request merged**